plugins/reporters/evaluated-model/src/funTest/resources/evaluated-model-reporter-test-deduplicate-expected-output.yml

```diff
@@ -1166,16 +1166,13 @@ statistics:
       MIT: 1
   execution_duration_in_seconds: 3125
 repository:
-  vcs:
-    type: ""
-    url: ""
-    revision: ""
-    path: ""
-  vcs_processed:
-    type: "Git"
-    url: "https://github.com/oss-review-toolkit/ort.git"
-    revision: "master"
-    path: "analyzer/src/funTest/assets/projects/synthetic/gradle/lib"
+  provenance:
+    vcs_info:
+      type: "Git"
+      url: "https://github.com/oss-review-toolkit/ort.git"
+      revision: "master"
+      path: "analyzer/src/funTest/assets/projects/synthetic/gradle/lib"
+    resolved_revision: "master"
```

Comment:
Something seems to be wrong here: A *resolved* revision should never be a symbolic name like "master".

Reply:
@sschuberth I'm working on a patch for to always have a resolved revision inside the `Repository`.

However, for the `EvaluatedModelReporterFunTests` specifically here:
They only have fixed input and result files, so the revisions would be hard coded anyway.
In the `ort` repo referenced, there is only a `main` branch, no `master`.
Do you want me to make up a revision?
Or should I change it it `main` and you the current revision?
(The file path referenced also does no longer exist)

Reply:
Nevermind, I found the revision inside the scanner results. my bad.